css/cart.css

```diff
@@ -0,0 +1,107 @@
+.btn-remove{
+    background-color: var(--white);
+    border:1px solid var(--light-red2);
+    color: var(--light-red2);
```

Comment:
Naming the color based on the component or functonality it is related to would be better. This will make it easier for you if you want to change the color in future.

Reply:
Ok. Will keep this in mind in my future PR.